Clinical-Genomics/scout
# Introduce spaces into long strings of comments that break the page view

Fix #2451.

We observed this before, but we had it fixed only in the report page, see https://github.com/Clinical-Genomics/scout/pull/1976/files

**How to test**:
1. Open a case and add this comment: 
`A,very,long,comment,with,no,spacesonlyaveryveryaveryveryaveryveryaveryveryaveryveryaveryveryaveryveryaveryveryaveryveryaveryveryaveryveryaveryveryaveryveryaveryverylong string. and, then,some,more,genes,then,some,more,genes,then,some,more,genes,then,some,more,genes,then,some,more,genes,then,some,more,genes.`
1. Case page should break on master
1. Case page should be OK using this branch
1. Test also by adding the same comment on a variant


**Expected outcome**:
The functionality should be working
Take a screenshot and attach or copy/paste the output.

**Review:**
